requirements/base.txt

```diff
@@ -8,7 +8,7 @@ asgiref==3.8.1
     # via django
 django==4.2.16
     # via
-    #   -c requirements/common_constraints.txt
+    #   -c /home/runner/work/edx-submissions/edx-submissions/requirements/common_constraints.txt
```

Comment:
FYI This is a known issue with pip-compile: https://github.com/jazzband/pip-tools/issues/2131